gui/vtkMaptkCamera.cxx

```diff
@@ -114,6 +114,61 @@ bool vtkMaptkCamera::ProjectPoint(kwiver::vital::vector_3d const& in,
   out[1] = ppos[1];
   return true;
 }
+/**
+  *
+  * WARNING: The convention here is that depth is NOT the distance between the
+  * camera center and the 3D point but the distance between the projection of
+  * the 3D point on the optical axis and the optical center.
+*/
+
+//-----------------------------------------------------------------------------
+bool vtkMaptkCamera::UnprojectPoint(double pixel[2], double depth,
+                                    kwiver::vital::vector_3d *unProjectedPoint)
+{
+  // Build camera matrix
+  auto const K = this->MaptkCamera->intrinsics()->as_matrix();
+  auto const KtriangularView = K.triangularView<Eigen::Upper>();
```

Comment:
You might consider using `this->MaptkCamera->intrinsics()->unmap()` instead of directly applying the inverse calibration matrix.  This should do the same thing, but will also account for radial distortion if there is any.  It would be something like this:

    auto const T = this->MaptkCamera->translation();
    auto const R = kwiver::vital::matrix_3x3d(this->MaptkCamera->rotation());
    kwiver::vital::vector_2d normPoint = this->MaptkCamera->intrinsics()->unmap(kwiver::vital::vector_2d(pixel[0], pixel[1]));
    kwiver::vital::vector_3d homogPoint(normPoint[0], normPoint[1], 1.0);
    *unProjectedPoint = R.transpose() * ((depth * homogPoint) - T);